content/en/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -101,6 +101,21 @@ across namespaces, you need to use the fully qualified domain name (FQDN).
 As a result, all namespace names must be valid
 [RFC 1123 DNS labels](/docs/concepts/overview/working-with-objects/names/#dns-label-names).
 
+{{< warning >}}
+By creating namespaces with the same name as [public top-level
+domains](https://data.iana.org/TLD/tlds-alpha-by-domain.txt), Services in these
+namespaces can have short DNS names that overlap with public DNS records.
+Workloads from any namespace performing a DNS lookup without a trailing dot will
```

Comment:
```suggestion
Workloads from any namespace performing a DNS lookup without a [trailing dot](https://datatracker.ietf.org/doc/html/rfc1034#page-8) will
```

Reply:
That's a nit ^ but having some context about what a trailing dot means could be helpful (because a proper DNS name has the root label at the end with a dot, I guess?)

Reply:
Yes! That's exactly that and the RFC precision about absolute and relative names is great!